.github/workflows/README.md

```diff
@@ -0,0 +1,38 @@
+# rust-bitcoin workflow notes
+
+We are attempting to run max 20 parallel jobs using GitHub actions (usage limit for free tier).
+
+ref: https://docs.github.com/en/actions/learn-github-actions/usage-limits-billing-and-administration
+
+The minimal/recent manifests are handled by CI (rust.yaml).
+
+Jobs (shell functions) that get run twice, once for each manifest:
+
+1+2  stable
+3+4  nightly
+5+6  msrv
```

Comment:
```suggestion
- 1+2  `stable`
- 3+4  `nightly`
- 5+6  `msrv`
```

If you leave like this, it will just concatenate the lines into a single paragraph.

Reply:
Oh sweet, I've wondered why that happened before. FTR I almost never look at rendered markdown, I mostly just read it in my editor so I appreciate you pointing out any habitual render-causing mistakes I make.